### PR TITLE
New keyboard 'Sentraq S65-Plus'

### PR DESCRIPTION
**Pull request by michaelzoech**
This patch adds support for the https://sentraq.com/collections/kits/products/s65-plus-diy-keyboard-kit PCB, with a default ANSI keymap as provided by Sentraq.

### CODE REVIEW

File: keyboards/s65_plus/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#include "s65_plus.h"
```
Comment (noroadsleft) [inline keyboards/s65_plus/keymaps/default/keymap.c]: Please change this to `#include QMK_KEYBOARD_H`.

File: keyboards/s65_plus/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   * │ F5  │ F6  │CAPSL│▒▒▒▒▒│  A  │  S  │  D  │  F  │  G  │  H  │  J  │  K  │  L  │  ;  │  '  │▒▒▒▒▒│ENTER│PG_DN│
+   * ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   * │ F7  │ F8  │▒▒▒▒▒│LSHFT│  Z  │  X  │  C  │  V  │  B  │  N  │  M  │  ,  │  .  │  /  │▒▒▒▒▒│RSHFT│ UP  │ END │
+   * ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   * │ F9  │ F10 │LCTRL│L_GUI│L_ALT│█████│█████│█████│ SPC │█████│█████│█████│R_ALT│ FN0 │RCTL │LEFT │DOWN │RIGHT│
+   * └─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┘
+   */
+
+  /* 0: ANSI qwerty */
+  [_BL] = ANSI_KEYMAP(
```
Comment (noroadsleft) [inline keyboards/s65_plus/keymaps/default/keymap.c]: Please change `ANSI_KEYMAP` to `LAYOUT_ansi`.

File: keyboards/s65_plus/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+   *  ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   *  │     │     │     │▒▒▒▒▒│     │     │     │     │     │     │     │     │     │     │     │▒▒▒▒▒│     │     │
+   *  ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   *  │     │     │▒▒▒▒▒│     │UGTOG│UGMOD│UGHUI│UGHUD│UGSAI│UGSAD│UGVAI│UGVAD│     │     │▒▒▒▒▒│     │VOLU │     │
+   *  ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   *  │     │     │     │     │     │█████│█████│█████│     │█████│█████│█████│     │     │     │PREV │VOLD │NEXT │
+   *  └─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┴─────┘
+   */
+
+  [_AL] = ANSI_KEYMAP(
```
Comment (noroadsleft) [inline keyboards/s65_plus/keymaps/default/keymap.c]: Please change `ANSI_KEYMAP` to `LAYOUT_ansi`.

File: keyboards/s65_plus/s65_plus.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#ifndef S60X_H
+#define S60X_H
+
+#include "quantum.h"
+
+#define ANSI_KEYMAP( \
```
Comment (noroadsleft) [inline keyboards/s65_plus/s65_plus.h]: Please change `ANSI_KEYMAP` to `LAYOUT_ansi`.

**Comment by michaelzoech** [comment]
Resolved review comments

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**